zib/StructureDefinitions/zib-LaboratoryTestResult.Specimen-BodyStructure.xml

```diff
@@ -0,0 +1,56 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-LaboratoryTestResult.Specimen-BodyStructure" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-LaboratoryTestResult.Specimen-BodyStructure" />
+  <name value="ZibLaboratoryTestResultSpecimenBodyStructure" />
+  <title value="zib LaboratoryTestResult.Specimen BodyStructure" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The body structure where the specimen material is obtained." />
+  <purpose value="This BodyStructure resource represents the AnatomicalLocation and Morphology concepts of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) LaboratoryTestResult v7.0 (2024)](https://zibs.nl/wiki/LaboratoryTestResult-v7.0(2024EN))." />
```

Comment:
This scentense could be improved. Somewhat stating along the lines that this profile is needed to enrich the AnatomicalLocation on BodyStructure with the mapping of Morphology of zib LaboratoryTestResult.